src/platform/packages/shared/kbn-scout/src/cli/run_tests.test.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { FlagsReader } from '@kbn/dev-cli-runner';
+import { ToolingLog } from '@kbn/tooling-log';
+import { runScoutPlaywrightConfig } from './run_tests';
+import { initLogsDir } from '@kbn/test';
+import { parseTestFlags, runTests } from '../playwright/runner';
+
+jest.mock('@kbn/test', () => ({
+  initLogsDir: jest.fn(),
+}));
+
+jest.mock('../playwright/runner', () => ({
+  parseTestFlags: jest.fn().mockResolvedValue({ logsDir: 'path/to/logs/directory' }),
+  runTests: jest.fn().mockResolvedValue(undefined),
+  runTestsFn: jest.fn().mockResolvedValue(undefined),
```

Comment:
This is a good example why I would like us to unify commands and underlying functions; we don't have `startServer` and `startServerFn`, so it is already getting confusing while searching the code

Reply:
I've removed `runTestsFn` in [8798f16](https://github.com/elastic/kibana/pull/216272/commits/8798f16a1a83bb38d3d4af731fe0de0cabc9d3cc) 🎉 tests still pass.